src/ol_openedx_sentry/settings/sentry.py

```diff
@@ -111,11 +108,10 @@ def plugin_settings(app_settings):
         sentry_sdk.init(
             dsn=sentry_dsn,
             environment=env_tokens.get("SENTRY_ENVIRONMENT"),
-            integrations=[DjangoIntegration(), CeleryIntegration(), RedisIntegration()],
```

Comment:
This line is not required with SDK v2 because all of these integrations will be enabled automatically.